### PR TITLE
CCS: Update ids and urls

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?

Update ids and url/page title to match those in the in the [spreadsheet](https://docs.google.com/spreadsheets/d/1BY1zRuVIL7ipsoVjg3V0fJ9vyaiWNH-AnsCC6O-LoKg/edit?usp=sharing). Use **CCS Household URL** and **CCS Individual URL** tabs. Changes are described on [this Trello card](https://trello.com/c/jpRV4KOu).

### How to review

Run quick launch schemas and check that urls are correct. Check id changes in jsonnet.

### Checklist

- [x] Jsonnet files conform to the latest [style guide](/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md)

#### England

- [CCS](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/change-ids-for-ccs/schemas/en/ccs_household_gb_eng.json)

#### Wales

- [CCS](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/change-ids-for-ccs/schemas/en/ccs_household_gb_wls.json)

### CODE REVIEW

**Comment by petechd** [comment]
[[I] Change IDs for CCS (M)](https://trello.com/c/jpRV4KOu/4009-i-change-ids-for-ccs-m)

File: source/jsonnet/england-wales/ccs/blocks/who-lives-here/any_more_people_living_here.jsonnet [hunk 1]
```diff
@@ -106,10 +106,10 @@ local editQuestion(questionTitle) = {
     ],
   },
   add_block: {
-    id: 'anyone-else-temp-away-add-person',
+    id: 'any-more-people-living-here-add-person',
     type: 'ListAddQuestion',
     question: {
-      id: 'anyone-else-temp-away-add-question',
+      id: 'any-more-people-living-here-add-question',
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/ccs/blocks/who-lives-here/any_more_people_living_here.jsonnet:112]: The question ids should use the full block id, so this should be:
```suggestion
      id: 'any-more-people-living-here-add-person-question',
```

This applies to a few places in this file.

**Comment by iwootten** [comment]
I'm not sure the ethnic group types need the file renames, I left them all as ethnic_group_{whatever}.jsonnet for eng/wls census as it seemed better that they would all be ordered together on the file system. Additionally seems like {whatever} is the first part of each id anyway.

**Comment by petechd** [comment]
> I'm not sure the ethnic group types need the file renames, I left them all as ethnic_group_{whatever}.jsonnet for eng/wls census as it seemed better that they would all be ordered together on the file system. Additionally seems like {whatever} is the first part of each id anyway.

I prefer to consequently use same ids/urls throughout the schema and no exceptions but it's up to reviewers.

**Comment by iwootten** [comment]
> I prefer to consequently use same ids/urls throughout the schema and no exceptions but it's up to reviewers.

Sure, but either way we should make sure they match across ccs/census.

**Comment by petechd** [comment]
> > I prefer to consequently use same ids/urls throughout the schema and no exceptions but it's up to reviewers.
> 
> Sure, but either way we should make sure they match across ccs/census. I'm a reviewer by the way 😄

In my defence: there are exceptions in ethnic_group files themselves, ethnic_group.jsonnet has same ids as the filename and the rest of ethic_group files has different ids than filenames.

**Comment by iwootten** [comment]
> "Simple is better than complex"😀.

Honestly, I don't mind either way. I felt that the filenames for ethnicity were unnecessarily verbose. What I'm saying is by changing them in this way it will require updating the existing Census ones for HH/HI to match. 

We don't have rigid rules on the filenaming conventions for jsonnet and should perhaps consider including it in the styleguide conventions.

**Comment by ajmaddaford** [comment]
@petechd @iwootten I agree with Ian on this one. The HH/HI ethnic groups were done as Ian has highlighted, and the old filenames means they get grouped together.

**Pull request merged**